Update classes.md (#1611)
JetBrains/kotlin-web-site: pages/docs/reference/classes.md

`Any` has three methods: `equals()`, `hashCode()` and `toString()`. Thus, they are defined for all Kotlin classes. 

> #### "open" Keyword

Reviewer: Please remove the heading. `open` is the synonym for inheritance in Kotlin, so we can introduce it right under the `Inheritance` heading.

> In Kotlin, all the classes are final by default i.e. they can’t be inherited.

Reviewer: ```suggestion
By default, Kotlin classes are final: they can’t be inherited.
```

> To make it inheritable, you must mark it with the keyword **open**. Otherwise it will throw error saying “type is final so can’t be inherited”. 

Reviewer: ```suggestion
To make a class inheritable, mark it with the `open` keyword.
```
We usually don't describe negative cases and errors in documentation.

>  *This is opposite to Java where you have to make your class final explicitly.*

Reviewer: Please remove this.
We deliberately avoid comparisons with Java in Kotlin docs, so that people with no Java background understand them better.
